web/html/src/manager/minion/packages/package.type.ts

```diff
@@ -1,12 +1,8 @@
-export type OptionalValue = {
```

Comment:
Looking at the rest of the code, it doesn't seem safe to me to remove this type and replace it with `number`. It seems we want `number | undefined` instead?